src/demoserver.cpp

```diff
@@ -233,9 +233,39 @@ void DemoServer::handle_packet(AOPacket packet)
             QString packet = "CT#DEMO#" + tr("min_wait is deprecated. Use the client Settings for minimum wait instead!") + "#1#%";
             client_sock->write(packet.toUtf8());
         }
+        else if (contents[1].startsWith("/debug"))
+        {
+            QStringList args = contents[1].split(" ");
+            if (args.size() > 1)
+            {
+              bool ok;
+              int toggle = args.at(1).toInt(&ok);
+              if (ok && (toggle == 0 || toggle == 1)) {
+                debug_mode = toggle == 1;
+                QString packet = "CT#DEMO#" + tr("Setting debug mode to ") + (debug_mode ? "true" : "false") + "#1#%";
+                client_sock->write(packet.toUtf8());
+                // Debug mode disabled?
+                if (!debug_mode) {
+                  // Reset the timer
+                  client_sock->write("TI#4#1#0#%");
+                  client_sock->write("TI#4#3#0#%");
+                }
+              }
+              else
+              {
+                QString packet = "CT#DEMO#" + tr("Valid values are 1 or 0!") + "#1#%";
+                client_sock->write(packet.toUtf8());
+              }
+            }
+            else
+            {
+              QString packet = "CT#DEMO#" + tr("Set debug mode using /debug 1 to enable, and /debug 0 to disable, which will use the fourth timer (TI#4) to show the remaining time until next demo line.") + "#1#%";
```

Comment:
```suggestion
              QString packet = "CT#DEMO#" + tr("Set debug mode using /debug 1 to enable, and /debug 0 to disable, which will use the fifth timer (TI#4) to show the remaining time until next demo line.") + "#1#%";
```
isn't index 4 the fifth timer

Reply:
oops